contracts/two-party-pol-holder/src/msg.rs

```diff
@@ -466,6 +466,8 @@ pub enum MigrateMsg {
         pool_address: Option<String>,
         ragequit_config: Box<Option<RagequitConfig>>,
         covenant_config: Box<Option<TwoPartyPolCovenantConfig>>,
+        // TODO: Do we want to allow this?
```

Comment:
good question.. I think yes? If both parties agree, it's changing the covenant terms just as any other update